src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/help-lightweight.html

```diff
@@ -2,6 +2,4 @@
     If selected, try to obtain the Pipeline script contents directly from the SCM without performing a full checkout.
     The advantage of this mode is its efficiency; however, you will not get any changelogs or polling based on the SCM.
     (If you use <code>checkout scm</code> during the build, this will populate the changelog and initialize polling.)
-    Also build parameters will not be substituted into SCM configuration in this mode.
-    Only selected SCM plugins support this mode.
```

Comment:
I think it is important to still note that not all SCMs that support lightweight checkouts, and that even the ones that do support lightweight checkout do not all support build parameters, so maybe change the wording to something like this instead:

```
Only selected SCM plugins support this mode.
Build parameters may be substituted into SCM configuration in this mode when running SCM API plugin X.Y.Z or newer. Not all SCM plugins support this feature.
```